src/components/WhatsNew/WhatsNew.tsx

```diff
@@ -0,0 +1,71 @@
+import { ModalContent, ModalHeader, Button, ModalPreview, nullable } from '@taskany/bricks';
+import { useCallback, useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { trpc } from '../../trpc/trpcClient';
+import { useLocale } from '../../hooks/useLocale';
+
+import { tr } from './WhatsNew.i18n';
+import s from './WhatsNew.module.css';
+
+export const WhatsNew = () => {
+    const locale = useLocale();
+    const [iframeReady, setIframeReady] = useState(false);
+    const [open, setOpen] = useState(false);
+
+    const { data } = trpc.whatsnew.check.useQuery({
+        locale,
```

Comment:
add `staleTime: Infinity`, we don't need to do this check more than once on page load